lib/screens/sentence_item.dart

```diff
@@ -1 +1,27 @@
-// view for item sentence
\ No newline at end of file
+import "package:flutter/material.dart";
+
+import "../models/sentence_model.dart";
+
+class SentenceItem extends StatelessWidget {
+  final Sentence _phrase;
+  final int _index;
+
+  SentenceItem(this._phrase, this._index);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.all(16.0),
```

Comment:
Is there some good practice for magic numbers?

Reply:
Will clean all of this later.